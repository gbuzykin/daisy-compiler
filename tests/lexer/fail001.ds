const s = "unterminated string constant