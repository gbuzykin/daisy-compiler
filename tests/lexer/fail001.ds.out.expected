./lexer/fail001.ds:1:1: debug: token
 1 | const u64_max = 0x10000000000000000u64;
   | ^~~~~
./lexer/fail001.ds:1:7: debug: id: u64_max
 1 | const u64_max = 0x10000000000000000u64;
   |       ^~~~~~~
./lexer/fail001.ds:1:15: debug: token
 1 | const u64_max = 0x10000000000000000u64;
   |               ^
./lexer/fail001.ds:1:17: error: too large integer literal
 1 | const u64_max = 0x10000000000000000u64;
   |                 ^~~~~~~~~~~~~~~~~~~~~~
./lexer/fail001.ds:1:17: debug: integer number: 0
 1 | const u64_max = 0x10000000000000000u64;
   |                 ^~~~~~~~~~~~~~~~~~~~~~
./lexer/fail001.ds:1:39: debug: token
 1 | const u64_max = 0x10000000000000000u64;
   |                                       ^
./lexer/fail001.ds: info: warnings 0, errors 1
